### PR TITLE
Add creating xml for externally launched virtiofsd

### PR DESCRIPTION
**Pull request by yafu-1**
Signed-off-by: Yan Fu <yafu@redhat.com>

### CODE REVIEW

File: virttest/utils_test/libvirt_device_utils.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
-import os
 import uuid
 
 from virttest.libvirt_xml.devices.filesystem import Filesystem
 
 
-def create_fs_xml(fsdev_dict):
+def create_fs_xml(fsdev_dict, launched_mode):
```
Comment (chunfuwen) [inline virttest/utils_test/libvirt_device_utils.py:6]: Do you want to set launched_mode as one default value:auto?

File: virttest/utils_test/libvirt_device_utils.py [hunk 2]
```diff
@@ -17,9 +17,9 @@ def create_fs_xml(fsdev_dict):
     target = fsdev_dict.get("target", {'dir': 'fs_mount'})
     binary_dict = fsdev_dict.get('binary', {})
 
-    not os.path.isdir(source['dir']) and os.mkdir(source['dir'])
```
Comment (kylazhang) [inline virttest/utils_test/libvirt_device_utils.py:20]: why remove this line here? no need to mkdir?

File: virttest/utils_test/libvirt_device_utils.py (hunk 1 above)
Comment (kylazhang) [reply virttest/utils_test/libvirt_device_utils.py:6]: Same concern here, do we need default value here?

**Review by kylazhang — COMMENTED** [review]
Others are ok for me

File: virttest/utils_test/libvirt_device_utils.py (hunk 1 above)
Comment (yafu-1) [reply virttest/utils_test/libvirt_device_utils.py]: Modified.

File: virttest/utils_test/libvirt_device_utils.py (hunk 2 above)
Comment (yafu-1) [reply virttest/utils_test/libvirt_device_utils.py:20]: The dir is created in tp-libvirt now.

**Review by kylazhang — APPROVED** [review]

**Review by Yingshun — APPROVED** [review]

**Comment by kylazhang** [comment]
As it is a little urgent and all reviewer's comments fixed, so just merge it now. Thanks

**Pull request merged**